### ISSUE TITLE
Error calculating scaling for radiobutton widget.

### ISSUE THREAD

**Issue by jtambasco** [matplotlib/matplotlib#4429]
Hi,

I believe there is a bug regarding the scaling of the radiobuttons if many radiobuttons are created.  If more radiobuttons are created on the same axis, they begin to overlap and become elipses.  Making the axis larger does not fix the problem.

This problem has been mentioned prior to this on SO (at least) twice:
http://stackoverflow.com/questions/16484528/matplotlib-widget-many-radiobuttons-issue/16484901?noredirect=1#comment48482822_16484901
http://stackoverflow.com/questions/23997920/matplotlib-radio-button-scaling

Code that replicates the problem and more screenshots can be found in the SO posts.

![screenshot from 2015-05-15 13-36-11](https://cloud.githubusercontent.com/assets/5669147/7646464/4f67c950-fb07-11e4-9b9f-d9a294770f68.png)

**Comment by petehuang** [comment]
Reconfirming with TkAgg, Windows 8.1, MPL 1.5.3, Python 2.7.12. Example code snippet here:

```
import matplotlib.pyplot as plt
from matplotlib.widgets import RadioButtons

rax = plt.axes([0.05, 0.1, 0.15, 0.7], axisbg='white')
radio3 = RadioButtons(rax, ('B1', 'B2', 'B3', 'B4', 'B5','B6', 'B7', 'B8', 'B9', 'B10'))

plt.show()
```

**Comment by QuLogic** [comment]
Fixed by #10780.
